src/__tests__/DependencyGraph-test.js

```diff
@@ -2119,6 +2119,86 @@ describe('DependencyGraph', function() {
     });
   });
 
+  describe('get sync dependencies on Windows', function() {
+    const realPlatform = process.platform;
+    beforeAll(function() {
```

Comment:
Woah, didn't even know that was a thing. We usually use `beforeEach` because we clean everything up at FB on every test. Mind doing this just so we stay consistent within the rest of the tests here? It makes it easier to understand tests I'd say :)